Src/WitsmlExplorer.Frontend/components/TopRightCornerMenu.tsx

```diff
@@ -39,6 +43,15 @@ const TopRightCornerMenu = (): React.ReactElement => {
     }
   }, []);
 
+  useEffect(() => {
+    const unsubscribeFromCredentialsEvents = CredentialsService.onCredentialStateChanged.subscribe(async (credentialState) => {
+      setLoginState({ isLoggedIn: credentialState.hasPassword, username: CredentialsService.getCredentials()[0]?.username, server: credentialState.server });
```

Comment:
`setUsername(CredentialsService.getCredentials()[0]?.username);` should be sufficient here as only the username is used.

Reply:
updated!